project/Dependencies.scala

```diff
@@ -40,7 +40,7 @@ object Dependencies {
     val JacksonDatatype = Jackson
     val JacksonDatabind = Jackson
     val Guava = "27.0-jre"
-    val Maven = "3.3.9"
+    val Maven = "3.6.0"
```

Comment:
Should this be `3.5.4`?

Reply:
Yeah, it should be `3.5.4` but considering the build passed using `3.6.0` I think we should go with `3.6.0`.